app/views/homes/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="home-page-main-container">
+  <div class="home-page-sub-container">
+    <h1>Welcome to the Recipe App</h1>
+    <div class="home-page-buttons-container">
+      <div class="public-recipes-page-button"><%= link_to 'Public Recipes Page', public_recipes_index_path %></div>
+      <div class="recipes-page-button"><%= link_to 'Recipes Page', recipes_index_path %></div>
+      <div class="inventories-page-button"><%= link_to 'Inventories page', inventories_index_path %></div>
+    </div>
+  </div>
+</div>
```

Comment:
According to the project requirements your app should have navigation menu that allows users to open all of the pages. It is difficult to navigate the app since there is no such menu. Kindly consider adding one to make your app more user-friendly. :+1:

Reply:
Could you please explain this point? I don't understand it

Reply:
The navigation menu (nav links) should be similar to something like the image shown below. You can click on them to go to different sections of the page. For instance you can have, the shopping list, recipes, foods, etc on the navigation menu such that any page you open will still have it on top for easy access and navigation. 

![Screenshot from 2023-08-20 01-48-15](https://github.com/sameedmohsin28/Recipe-App/assets/65233860/8e4b8fcf-3d45-4744-960e-499d493893da)